docs/source/ngv.rst

```diff
@@ -0,0 +1,20 @@
+Neuro-Glia-Vasculature (NGV)
+============================
+
+.. image:: img/ngv.drawio.svg
+   :alt: NGV Diagram
+   :align: center
+
+The NGV simulation requires three node populations, each stored in separate files: **neocortex_neurons**, **astrocytes**, and **vasculature**.
+
+- **Neocortex_neurons** represent standard neurons.
+- **Astrocytes** represent glial cells.
+- The **vasculature** file is unique: although it is structured and treated as a node file by Neurodamus, each node (row in the table) actually refers to a vasculature segment. The vasculature itself is divided into morphology nodes (used internally for structure reconstruction), sections (optional), and segments. For more details, refer to the official documentation:  
```

Comment:
FYI, the **vasculature** file is defined as a node file by the SONATA convention. Neurodamus doesn't create any object with this file, only reads its attributes for the relevant GlioVascular connections.  https://github.com/openbraininstitute/neurodamus/blob/ec61ef97bc866f03eddf9986246eb92b53dca4c7/neurodamus/io/sonata_config.py#L329
I don't know how to fit the information into your text. You can decide whether it is useful or not in the docs.

Reply:
let me figure something out. Thx!